### ISSUE TITLE
Fix compilation issue with openssl-1.1

### ISSUE THREAD

**Issue by ghost** [haiwen/ccnet-server#9]
In order to build ccnet-server with openssl-1.1 would it be possible to apply in the master branch the same commit as in ccnet (https://github.com/haiwen/ccnet/commit/445b72e52895431d02f5f6380a773409c689d8ea).
Thanks.

**Comment by theraw** [comment]
@eolianoe ~~can you please let me know what version of openssl do i need to build this? because i tried 1.1 failed, i also tried 1.0.1 it also failed but there are alot of openssl version so if you know which one should i use can you please let me know that?~~

Just tried to build from your fork works very well.  Thanks

**Comment by ghost** [comment]
Any news about this on this point ?

**Comment by niol** [comment]
I've been happily using this for a few months on my small 2-user setup.

**Comment by ghost** [comment]
Yes, it's working for all the users of the AUR but I'm waiting for a merge in the official repos by the devs....

**Comment by TCB13** [comment]
+1 on this. Just merge this already. It complies fine with this patch.
